documentation/recovery.md

```diff
@@ -102,11 +128,11 @@ On _\<working-postgres-server-fqdn\>_:
 
         systemctl stop puppet
 
-2. Add this line to /opt/puppetlabs/server/data/postgresql/11/data/pg\_ident.conf
+2. Add this line to /opt/puppetlabs/server/data/postgresql/14/data/pg\_ident.conf
```

Comment:
Add this line to /opt/puppetlabs/server/data/postgresql/<postgres_version>/data/pg_ident.conf
In the latest PE release 2023.8.0the PostgreSQL version is 14 /opt/puppetlabs/server/data/postgresql/14/data/pg_ident.confhttps://www.puppet.com/docs/pe/latest/component_versions_in_recent_pe_releases.html